samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java

```diff
@@ -148,6 +148,41 @@ public void assignMFAGroupBeforeScenario(Scenario scenario) {
 		groupList.forEach(group -> Page.getUser().addToGroup(group.getId()));
 	}
 
+	@Before("@requireWebAuthnRequiredGroupsForUser")
+	public void assignWebAuthnRequiredBeforeScenario(Scenario scenario) {
+		Assert.assertNotNull(Page.getUser());
+		List<String> groups = new ArrayList<>();
+		groups.add("WebAuthn Required");
+
+		List<Group> groupList = client.listGroups()
+				.stream()
+				.filter(group -> groups.contains(group.getProfile().getName()))
+				.collect(Collectors.toList());
+		Assert.assertFalse(groupList.isEmpty());
+		groupList.forEach(group -> Page.getUser().addToGroup(group.getId()));
+	}
+	public void assignWebAuthnGroup() {
+		if (Page.getA18NProfile() != null) {
+			logger.info("Searching for a user to be added to WebAuthn Required group: " + Page.getA18NProfile().getEmailAddress());
+			Optional<User> userToAdd = client.listUsers(Page.getA18NProfile().getEmailAddress(), null, null, null, null)
+					.stream().filter(x -> x.getProfile().getEmail().equals(Page.getA18NProfile().getEmailAddress())).findFirst();
+			if (userToAdd.isPresent()) {
+				List<String> groups = new ArrayList<>();
+				groups.add("WebAuthn Required");
```

Comment:
Make "WebAuthn Required" string into a static variable like `WEBAUTHN_GROUP_NAME`. Easier to read and also this string is used in multiple places

Reply:
updated the suggested change